### DISCUSSION TITLE
list-sessions always shows [Created 0s ago]

### DISCUSSION THREAD

**Discussion by steveclarke** [zellij-org/zellij#3008] [Q&A]
Whenever I want to restore a session and do a `zellij list-sessions` it returns a list of sessions with every one showing "Created 0s ago". This leaves me unable to ascertain which session I just closed. Is this normal behaviour or is it just me?

![image](https://github.com/zellij-org/zellij/assets/1305/53bb194e-c32d-473b-8177-f9840f2af7d1)

**Comment by imsnif** [comment]
This most probably happens because for some reason we can't get metadata about files in your filesystem. Could be a permission issue for the cache, or some sort of filesystem limitation? Just guessing...

**Reply by steveclarke** [reply]
I found out what the likely culprit is. I was running it from the downloaded app image. After I installed it from Homebrew on Linux it now shows the created time properly.

**Reply by huylv** [reply]
From my experience, it always shows the "Created 0s ago" statuses with the brebuilt binary. I already tried on Ubuntu and Redhat, same results. Maybe there is a problem with the binary? I would love to have this working because I cannot freely install everything on my work machine, although I do have root permission.

**Comment by airtonix** [comment]
<img width="561" height="438" alt="image" src="https://github.com/user-attachments/assets/e6321114-16a9-4658-836a-7ac0e56e4301" />

I installed via the release page: 

```
mise use -g --pin ubi:zellij-org/zellij
```

or you can even 

```
mise use -g --pin rust@latest
sudo dnf install perl-IPC-Cmd
mise use -g --pin cargo:zellij
```

brew is heresy when mise or asdf exists

**Reply by airtonix** [reply]
And so when installing zellij via cargo we get: 

<img width="719" height="327" alt="image" src="https://github.com/user-attachments/assets/2765139f-f7b1-428b-868b-471c3c67a5ac" />
